model/patch_lifecycle.go

```diff
@@ -162,16 +161,12 @@ func GetPatchedProject(ctx context.Context, p *patch.Patch, githubOauthToken str
 			return nil, "", errors.Wrapf(err, "Could not patch remote configuration file")
 		}
 	}
-	pp, _, err := LoadProjectInto(ctx, projectFileBytes, opts, p.Project, project)
+	_, _, err = LoadProjectInto(ctx, projectFileBytes, opts, p.Project, project)
```

Comment:
The reason we use the parser project output is because it contains the included files, so using projectFileBytes potentially omits a lot of new stuff. I'm not super sure what to do about that for this project; maybe patches need a new variable to store the PatchedProjectConfig (for lack of better term) if it exists. You could also just drop a TODO here and handle it in a new ticket.

Reply:
Added todo covering new ticket: https://jira.mongodb.org/browse/EVG-16000